include/tvm/ir_pass.h

```diff
@@ -181,11 +181,11 @@ Stmt Inline(Stmt stmt,
  * \param extern_buffer Map specifies external
  *    buffer assignment of input and outputs.
  * \param cache_line_size The size of CPU cache line.
+ * \param create_bound_attribute Whether to create bound attributes.
  * \return Transformed stmt.
  */
-Stmt StorageFlatten(Stmt stmt,
-                    Map<Tensor, Buffer> extern_buffer,
-                    int cache_line_size);
+Stmt StorageFlatten(Stmt stmt, Map<Tensor, Buffer> extern_buffer,
```

Comment:
nit, one line per argument

Reply:
thanks, fixed. BTW should TVM has its own .clang-format file ?

Reply:
There is a proposal in #1732 , we generally use Google C style, but haven't yet have our own .clang-format